src/main/kotlin/no/nav/syfo/client/KuhrSarClient.kt

```diff
@@ -180,13 +204,13 @@ fun samhandlerMatchingPaaOrganisjonsNavn(samhandlere: List<Samhandler>, orgName:
     }
 }
 
-fun filtererBortSamhanlderPraksiserPaaProsentMatch(
+fun samhandlerPraksisMatchTest(
     samhandlerPraksis: SamhandlerPraksisMatch?,
-    prosentMatch: Double,
+    percentageMatchLimit: Double,
     orgName: String,
     loggingMeta: LoggingMeta
 ): SamhandlerPraksisMatch? {
-    return if (samhandlerPraksis != null && samhandlerPraksis.percentageMatch >= prosentMatch) {
+    return if (samhandlerPraksis != null && samhandlerPraksis.percentageMatch >= percentageMatchLimit) {
```

Comment:
Går det an å gjøre sånn her
`return if (samhandlerPraksis?.percentageMatch >= percentageMatchLimit) {...}`?

Reply:
Kompilatoren liker ikke det: 
Operator call corresponds to a dot-qualified call 'samhandlerPraksis?.percentageMatch.compareTo(percentageMatchLimit)' which is not allowed on a nullable receiver 'samhandlerPraksis?.percentageMatch'.

Reply:
Ja, for med spørsmålstegn kan man må `null >= limit`, og det liker den ikke, det gir mening 👍 
https://stackoverflow.com/questions/46642191/kotlin-cannot-apply-two-conditionals-at-once-check-with-return-body-function